docs/docker/test.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+# This script requires the local Docker image (playwright:localbuild) by the build.sh
+
+set -ex
+
+CONTAINER_ID="$(docker run --name playwright-docker-test -d -t playwright:localbuild /bin/bash)"
+
+# Reason for using a separate Git repo inside the Docker container instead of the current one is that 'docker cp'
+# Does not respect the .dockerignore or .gitignore
+docker cp $(pwd)/playwright-clean-state "${CONTAINER_ID}":/home/pwuser/playwright
+docker exec -u root "${CONTAINER_ID}" chown -R pwuser /home/pwuser/playwright/
+
+# Skip Electron tests
+docker exec --workdir /home/pwuser/playwright/ "${CONTAINER_ID}" rm -rf test/electron/
+
+docker exec --workdir /home/pwuser/playwright/ "${CONTAINER_ID}" npm ci
+docker exec --workdir /home/pwuser/playwright/ "${CONTAINER_ID}" npm run build
+docker exec --workdir /home/pwuser/playwright/ "${CONTAINER_ID}" xvfb-run --auto-servernum --server-args="-screen 0 1280x960x24" -- bash -c "ulimit -c unlimited && node test-runner/cli test/ --jobs=1 --forbid-only --timeout=30000 --retries=3 --reporter=dot"
```

Comment:
let's drop the `ulimit -c unlimited` part from here (and you'll be able to drop `bash -c` as well)